tasks/Restaurant.tex

```diff
@@ -0,0 +1,90 @@
+\end{enumerate}
+
+\subsection{Setup}
+\begin{itemize}[nosep]
+	\item \textbf{Location:} A real restaurant fully equipped with a \enquote{Professional Barman} i.e.~the operator and at least three tables with \enquote{Professional Clients}.
+    \item \textbf{Kitchen-bar:} The \textit{Kitchen-bar} will be a table located at the restaurant's kitchen, next to the place where the robot started. It has the following setup.
+	\begin{itemize}[nosep]
+		\item \textbf{Barman:} A \textit{Professional Barman} (member of the TC) will be at the other side of the \textit{Kitchen-bar} to take the order provided by the robot and serve it in the official tray.
+		\item \textbf{Items:} Items will be located on the Kitchen-bar next to the \textit{Professional Barman}.
+	\end{itemize}
+\end{itemize}
+
+
+\subsection{Additional rules and remarks}
+\begin{enumerate}[nosep]
+	\item \textbf{Safety!} This test takes place in a public area. That is, there may be people standing, sitting or walking around the area throughout the test. The robot is expected to not even slightly touch anything and is immediately stopped in case of danger.
+
+	\item \textbf{Referees and guidance:} For safety reasons, the referees in this test are TC members. One of the referees follows the robot and is always in reach of the emergency button.
+
+    \item \textbf{Deus ex Machina:} Score reduction for requesting human assistance is applied per order as follows.
+        \begin{itemize}[nosep]
+          \item Needing guidance during the deliver phase causes a reduction of 40\% in scoring.
+          \item Needing assistance for picking up an item/the tray causes a reduction of 10\% in scoring.
+          \item Have a human place the item in a basket causes a reduction of 20\% in scoring.
+          \item Have the quest take the item out of the basket causes a reduction of 20\% in scoring.
+          \item Asking the barman at which side the \textit{Kitchen-bar} is causes a reduction of 5\% in the total scoring.
+        \end{itemize}
+
+	\item \textbf{Start:} There is no fixed start signal in this test, it starts when both robots are ready (though within a reasonable time).
+
+	\item \textbf{Order:} The way the user provides information to the robot is up to the robot's team. A natural interaction is preferred.
+
+	\item \textbf{Location:} This test can be arranged in any real restaurant or shopping mall. If this is not possible, the test can be conducted in an arbitrary room containing the appropriate locations.
+	  The only requirement is that this room is not part of the arena and that the teams do not know the room beforehand.
+	  The exact location, including the object and delivery locations, will be defined by the technical committee on site (and in corporation with the local organization).
+	  In addition, to avoid unnecessary time investment for navigation, the distances between tables and the \enquote{Kitchen Bar} will be minimal.
```

Comment:
Suggest adding: `nearby electrical outlets, and wi-fi access should not be expected`

Reply:
Also: something specifying that `the the location will likely be space confined. All RoboCup@Home participants will be expected to follow the instructions of the onsite OC, and may not be able to watch their or other teams performances.`

Reply:
Wifi cant be expected is covered in the general rules. I think this is fine as is and agree with Matthijs that we should not overcomplicate the rules in the test description.